ginga/main.py

```diff
@@ -292,10 +297,10 @@ def main(self, options, args):
 
         if toolkit == 'choose':
             try:
-                from ginga.qtw import QtHelp
+                from ginga.qtw import QtHelp  # noqa
```

Comment:
It's just an incantation to tell the PEP8 checker that I know it's not PEP8 compliant and stop complaining. :)